<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>412</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>11</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Mine Sweeper</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/Images/icon.ico</normaloff>:/Images/icon.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="pCentralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="pMainLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="pGameInfoLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QLabel" name="pMineLabel">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>42</height>
         </size>
        </property>
        <property name="text">
         <string>MINES:999</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pTimeLabel">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>42</height>
         </size>
        </property>
        <property name="text">
         <string>TIME:999</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="GraphicsWidget" name="pGraphics" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>344</width>
        <height>344</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="pMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>344</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="pMenuStart">
    <property name="title">
     <string>Start(S)</string>
    </property>
    <addaction name="pActionEasyLevel"/>
    <addaction name="pActionMediumLevel"/>
    <addaction name="pActionHardLevel"/>
    <addaction name="pActionCustomLevel"/>
    <addaction name="separator"/>
    <addaction name="pActionPause"/>
    <addaction name="pActionRestart"/>
   </widget>
   <widget class="QMenu" name="pMenuHelp">
    <property name="title">
     <string>Help(H)</string>
    </property>
    <addaction name="pActionRecord"/>
    <addaction name="separator"/>
    <addaction name="pActionAbout"/>
   </widget>
   <addaction name="pMenuStart"/>
   <addaction name="pMenuHelp"/>
  </widget>
  <action name="pActionEasyLevel">
   <property name="text">
    <string>Easy(E)</string>
   </property>
  </action>
  <action name="pActionMediumLevel">
   <property name="text">
    <string>Medium(M)</string>
   </property>
  </action>
  <action name="pActionHardLevel">
   <property name="text">
    <string>Hard(H)</string>
   </property>
  </action>
  <action name="pActionCustomLevel">
   <property name="text">
    <string>Custom(C)</string>
   </property>
  </action>
  <action name="pActionPause">
   <property name="text">
    <string>Pause(P)</string>
   </property>
  </action>
  <action name="pActionRestart">
   <property name="text">
    <string>Restart(R)</string>
   </property>
  </action>
  <action name="pActionRecord">
   <property name="text">
    <string>Record(R)</string>
   </property>
  </action>
  <action name="pActionAbout">
   <property name="text">
    <string>About(A)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsWidget</class>
   <extends>QWidget</extends>
   <header location="global">Graphics.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
